transformations/whitespace_perturbation/transformation.py

```diff
@@ -0,0 +1,46 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def whitespace(text, remove_prob=0.1, add_prob=0.05, seed=0, max_outputs=1):
+    random.seed(seed)
+    perturbed_texts = []
+    for _ in range(max_outputs):
+        perturbed_text = []
+        for char in text:
```

Comment:
Could you move this loop and the outer one to the `generate` function? This is so the whitespace function would operate at the character level and will make it easier for us to attach it to a general adversarial search function. Thanks! Otherwise the PR looks good (to me).

Reply:
done!